gapic/src/main/com/google/gapid/perfetto/TraceView.java

```diff
@@ -72,7 +72,7 @@ public TraceView(Composite parent, Models models, Widgets widgets) {
 
   private static TraceComposite<State.ForSystemTrace> createTraceUi(
       Composite parent, Models models, Theme theme) {
-    return new TraceComposite<State.ForSystemTrace>(parent, models.analytics, theme) {
+    return new TraceComposite<State.ForSystemTrace>(parent, models, theme) {
```

Comment:
Models contains quite a lot things, considering the main extra thing you need in this PR is models.perfetto, I would suggest only passing this extra object in for now.